pkgs/top-level/all-packages.nix

```diff
@@ -37665,8 +37665,7 @@ with pkgs;
   xzoom = callPackage ../tools/X11/xzoom {};
 
   yabai = darwin.apple_sdk_11_0.callPackage ../os-specific/darwin/yabai {
-    inherit (darwin.apple_sdk.frameworks) Cocoa Carbon ScriptingBridge;
-    inherit (darwin.apple_sdk_11_0.frameworks) SkyLight;
```

Comment:
The syntax before was a bit sketchy. Don't mix SDK versions.

Reply:
Yeah, that definitely wasn't the best. I did so because I used regular `pkgs.callPackage` when creating the previous PR, with which syntax errors are thrown when all frameworks are included from `apple_sdk_11`. In a review comment you suggested to use `apple_sdk_11_0.callPackage`, using that fixes the issue. I never thought to go back and check if the mixture was still required.


Here is the log, it goes on for quite a while with similar errors. I'm not sure why it happens.
```
yabai> /nix/store/33am5aq9p2vb2m8f7g4pz32jxf25gxnm-apple-framework-CoreFoundation-11.0.0/Library/Frameworks/CoreFoundation.framework/Headers/CFURL.h:1189:83: error: expected ','
yabai>     kCFURLBookmarkCreationWithSecurityScope API_AVAILABLE(macos(10.7), macCatalyst(13.0))  API_UNAVAILABLE(ios, watchos, tvos) = ( 1UL << 11 ), // Mac OS X 10.7.3 and later, include information in the bookmark data which allows the same sandboxed process to access the resource after being relaunched
yabai>                                                                                   ^
yabai> In file included from ./src/osax/loader.m:2:
yabai> In file included from ./src/osax/x86_64/loader.m:1:
yabai> In file included from /nix/store/qlri4s3662flhy1y3lzq8j1hpvnf8xnp-apple-framework-Foundation-11.0.0/Library/Frameworks/Foundation.framework/Headers/Foundation.h:/nix/store/33am5aq9p2vb2m8f7g4pz32jxf25gxnm-apple-framework-CoreFoundation-11.0.0/Library/Frameworks/CoreFoundation.framework/Headers/CFURL.h:1190:98: 6error: expected ','
yabai> :
yabai> In file included from /nix/store/33am5aq9p2vb2m8f7g4pz32jxf25gxnm-apple-framework-CoreFoundation-11.0.0/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:60:
yabai> In file included from /nix/store/33am5aq9p2vb2m8f7g4pz32jxf25gxnm-apple-framework-CoreFoundation-11.0.0/Library/Frameworks/CoreFoundation.framework/Headers/CFPropertyList.h:17:
yabai> In file included from /nix/store/33am5aq9p2vb2m8f7g4pz32jxf25gxnm-apple-framework-CoreFoundation-11.0.0/Library/Frameworks/CoreFoundation.framework/Headers/CFStream.h:16:
yabai> /nix/store/33am5aq9p2vb2m8f7g4pz32jxf25gxnm-apple-framework-CoreFoundation-11.0.0/Library/Frameworks/CoreFoundation.framework/Headers/CFURL.h    kCFURLBookmarkCreationSecurityScopeAllowOnlyReadAccess API_AVAILABLE(macos(10.7), macCatalyst(13.0)) API_UNAVAILABLE(ios, watchos, tvos) = ( 1UL << 12 ), // Mac OS X 10.7.3 and later, if used with kCFURLBookmarkCreationWithSecurityScope, at resolution time only read access to the resource will be granted:1189
yabai> :83: error:                                                                                                  ^
yabai> expected ','
```